api-test/src/main/resources/pms/DeactivateMISPPartner/DeactivateMISPPartner.yml

```
@@ -0,0 +1,17 @@
+DeactivateMISPPartner:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)